packages/nimble-components/src/select/tests/select-opened-matrix.stories.ts

```diff
@@ -45,29 +49,109 @@ if (remaining.length > 0) {
+export const selectBelowOpenNoFilterLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.none),
+    lightThemeWhiteBackground
+);
+
+export const selectBelowOpenStandardFilterLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard),
+    lightThemeWhiteBackground
+);
+
+export const selectAboveOpenNoFilterLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[1], FilterMode.none),
     lightThemeWhiteBackground
 );
-export const selectAboveOpenLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
-    component(positionStates[1]),
+
+export const selectAboveOpenStandardFilterLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[1], FilterMode.standard),
     lightThemeWhiteBackground
 );
 
-export const selectBelowOpenColorThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
-    component(positionStates[0]),
+export const selectBelowOpenColorNoFilterThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.none),
+    colorThemeDarkGreenBackground
+);
+
+export const selectBelowOpenColorStandardFilterThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard),
+    colorThemeDarkGreenBackground
+);
+
+export const selectAboveOpenNoFilterColorThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[1], FilterMode.none),
     colorThemeDarkGreenBackground
 );
-export const selectAboveOpenColorThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
-    component(positionStates[1]),
+
+export const selectAboveOpenStandardFilterColorThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[1], FilterMode.standard),
     colorThemeDarkGreenBackground
 );
 
-export const selectBelowOpenDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
-    component(positionStates[0]),
+export const selectBelowOpenNoFilterDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.none),
+    darkThemeBlackBackground
+);
+
+export const selectBelowOpenStandardFilterDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard),
+    darkThemeBlackBackground
+);
+
+export const selectAboveOpenNoFilterDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[1], FilterMode.none),
+    darkThemeBlackBackground
+);
+
+export const selectAboveOpenStandardFilterDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[1], FilterMode.standard),
     darkThemeBlackBackground
 );
-export const selectAboveOpenDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
-    component(positionStates[1]),
+
+const badFilterPlayFunction = (): void => {
```

Comment:
nit: This isn't really a "bad" filter (there isn't anything invalid or incorrect about it). What you called it something like "noMatchesFilterPlayFunction" instead?